<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Polygen Painter</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="toolBarWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>171</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#toolBarWidget
{
	background: url(:/images/6.jpg);
}</string>
    </property>
    <widget class="QToolBox" name="toolBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>591</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Buxton Sketch</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="color">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>151</width>
        <height>517</height>
       </rect>
      </property>
      <attribute name="label">
       <string> Color and Painting</string>
      </attribute>
      <widget class="QColumnView" name="backGround">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>48</width>
         <height>48</height>
        </rect>
       </property>
      </widget>
      <widget class="MyButton" name="green">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>160</y>
         <width>32</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#green
{
	background-image: url(:/images/green.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="MyButton" name="yellow">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>32</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#yellow
{
	background-image: url(:/images/yellow.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="MyButton" name="blueLight">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>160</y>
         <width>32</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#blueLight
{
	background-image: url(:/images/blue-light.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="MyButton" name="greenLight">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>120</y>
         <width>32</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#greenLight
{
	background-image: url(:/images/green-light.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="MyButton" name="purple">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>32</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#purple
{
	background-image: url(:/images/purple.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="MyButton" name="purpleLight">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>120</y>
         <width>32</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#purpleLight
{
	background-image: url(:/images/purple-light.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="MyButton" name="red">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>32</width>
         <height>32</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#red
{
	background-image: url(:/images/red.png);
    
	border-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QColumnView" name="frontGround">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>48</width>
         <height>48</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255,127,39);</string>
       </property>
      </widget>
      <widget class="MyButton" name="blue">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>32</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#blue
{
	background-image: url(:/images/blue.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
      <widget class="MyButton" name="orange">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>32</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#orange
{
	background-image: url(:/images/orange.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="polygen">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>270</y>
         <width>52</width>
         <height>52</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>隶书</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#polygen
{
	background-image: url(:/images/Free-Transform-48.png)
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="fill">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>350</y>
         <width>52</width>
         <height>52</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#fill
{
	background-image: url(:/images/Paint-Bucket-48.png)
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="circle">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>350</y>
         <width>52</width>
         <height>52</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#circle
{
	background-image: url(:/images/Shape62-48.png)
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="square">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>52</width>
         <height>52</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>华文隶书</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#square
{
	background-image: url(:/images/Selection-48.png)
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Buxton Sketch</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>with inner circle</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>320</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>长方形</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>320</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>多边形</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>410</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>椭圆</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>410</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>油漆桶</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tools">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>151</width>
        <height>517</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <attribute name="label">
       <string>           Tools</string>
      </attribute>
      <widget class="QPushButton" name="rotateLeft">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>52</width>
         <height>52</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#rotateLeft
{
	background-image: url(:/images/Command-Undo-48.png)
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="rotateRight">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>10</y>
         <width>52</width>
         <height>52</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#rotateRight
{
	background-image: url(:/images/Command-Redo-48.png)
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="reflectDown">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>52</width>
         <height>52</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#reflectDown
{
	background-image: url(:/images/Share-02-48.png)
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="MyButton" name="zoom">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>250</y>
         <width>52</width>
         <height>52</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#zoom
{
	background-image: url(:/images/Zooming-48.png)
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="clip">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>52</width>
         <height>52</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#clip
{
	background-image: url(:/images/Cut-48 (1).png)
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="reflectRight">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>90</y>
         <width>52</width>
         <height>52</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#reflectRight
{
	background-image: url(:/images/Turn-48.png)
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="moveLeft">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>52</width>
         <height>52</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#moveLeft
{
	background-image:url(:/images/Swap-Left-48.png)
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="moveRight">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>170</y>
         <width>52</width>
         <height>52</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#moveRight
{
	background-image: url(:/images/Swap-Right-48.png)
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>右转90</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>左转90</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>上下翻转</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>140</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>左右翻转</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>220</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>左移</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>220</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>右移</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>以矩形裁剪</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>310</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>缩放</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>410</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>以多边形裁剪</string>
       </property>
      </widget>
      <widget class="QPushButton" name="clipPolygen">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>350</y>
         <width>52</width>
         <height>52</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#clipPolygen
{
	background-image: url(:/images/Cut-48 (1).png)
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="PaintWidget" name="paintArea" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>-1</y>
      <width>681</width>
      <height>621</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>681</width>
      <height>621</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>681</width>
      <height>621</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>华文新魏</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyButton</class>
   <extends>QPushButton</extends>
   <header>mybutton.h</header>
  </customwidget>
  <customwidget>
   <class>PaintWidget</class>
   <extends>QWidget</extends>
   <header>paintwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
